src/NuGet.Clients/NuGet.CommandLine/Commands/SourcesCommand.cs

```diff
@@ -78,6 +78,10 @@ public override void ExecuteCommand()
                     EnableSourceRunner.Run(enableSourceArgs, () => Console);
                     break;
                 case SourcesAction.List:
+                    if (Format == SourcesListFormat.None)
```

Comment:
Was the return code a 0?

Reply:
no, return code was 1...and message was red, etc...